script/Script.py

```diff
@@ -0,0 +1,15 @@
+import MonthCalculation
+import datetime
+import GraphQLQuery
+import Log
+import os
+
+
+def main():
+    Log.clearLog()
+    today = datetime.datetime.now()
+    for i in range(0, os.environ['MONTH']):
```

Comment:
I don't think this will work, since `os.environ['MONTH']` returns a string, not an integer.

Reply:
```suggestion
    for i in range(0, int(os.environ['MONTH'])):
```